extensions/wikia/VideoHandlers/apiwrappers/AnyclipApiWrapper.class.php

```diff
@@ -3,7 +3,7 @@
 class AnyclipApiWrapper extends ApiWrapper {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Class "AnyclipApiWrapper" has 28 methods, which is greater than 20 authorized. Split it into smaller classes. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1448)

[project-id]: wikia:app